cypress/integration/add_dog_view.js

```diff
@@ -1,11 +1,50 @@
 describe('Add Dog View', () => {
 
   beforeEach(() => {
-    
+    cy.loadHome()
+      .get('.add-dog-nav')
+      .click()
   });
 
-  it('Test Description', () => {
-    
+  it('Should have the header with a title, and 4 links', () => {
+    cy.get('.title').should('include.text', 'Pawty Trainer')
+      .get('.dashboard-nav').should('include.text', 'Dashboard')
+      .get('.add-dog-nav').should('include.text', 'Add Dog')
+      .get('.add-event-nav').should('include.text', 'Add Event')
+      .get('.calendar-nav').should('include.text', 'Calendar')
   });
 
+  it('Should have a form with a title, 3 inputs, and a button', () => {
+    cy.get('.train-dog-title').should('include.text', 'Train your dog!')
```

Comment:
This one is failing because:
 + expected - actual

      -'Train Your Dog!'
      +'Train your dog!'